src/api/app/views/webui2/webui/projects/maintenance_incidents/index.html.haml

```diff
@@ -0,0 +1,35 @@
+- @pagetitle = 'Maintenance Incidents'
+
+.card.mb-3
+  = render partial: 'tabs', locals: { project: @project }
+  .card-body
+    %h3
+      Maintenance incidents
+      %span.badge.badge-primary
+        = @incidents.count
+
+    %table.w-100.responsive.table.table-sm.table-striped.table-bordered#incident-table
+      %thead
+        %tr
+          %th
+            Summary
+          %th
+            Category
+          %th
+            Packages
+          %th
+            Info
+          %th
+            Release Targets
+          %th
+            Build
+      %tbody
+        = render partial: 'webui/projects/maintenance_incidents/incident_table_entry', collection: @incidents
```

Comment:
I think that's a good idea. That's also the one of the TODOs of the this [card](https://trello.com/c/4hbIZuK9/1095-maintenance-incidents). I just find it nicer to do this in smaller steps. And because some of the core users of this page need to be able to see all incidents at once and don't want to click on the pagination counter after every page load, I wanted to research first how we can store the counter for the page entries first.

Reply:
so you want to merge this PR and then after that work on the server side pagination @bgeuken ?

Reply:
I wouldn't have seen a problem with that. At least I didn't want to do all changes in one go. This way I managed to already understand the general structure of this table and find possible bugs I introduced early on.

Reply:
It's ok for me, I just wanted to make sure that we don't forget it!